templates/slurm.conf.j2

```diff
@@ -46,7 +46,9 @@ ProctrackType=proctrack/pgid
 #TreeWidth=50
 #TmpFS=
 #UsePAM=
-#
+{% if openhpc_slurm_configless %}SlurmctldParameters=enable_configless{% endif %}
+
```

Comment:
Was this white-space intentional?

Reply:
Yes - ensures you get a blank line either way, else the following # gets appended to the SlurmctldParameters .. line